.github/workflows/industrial_ci_action.yml

```diff
@@ -9,48 +9,41 @@ env:
   CMAKE_ARGS: "-DCATKIN_ENABLE_CLANG_TIDY=true"
 
 jobs:
-  build_test_main:
-    name: "Build + Test with Main Repo of melodic (http://packages.ros.org/ros/ubuntu)"
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v1
-      - uses: 'ros-industrial/industrial_ci@master'
+  builds:
+    strategy:
+      matrix:
         env:
-          ROS_REPO: main
-          ROS_DISTRO: melodic
-    continue-on-error: true
-
-  build_test_testing:
-    name: "Build + Test with Testing Repo of melodic (http://packages.ros.org/ros-testing/ubuntu)"
-    runs-on: ubuntu-18.04
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
+          - {ROS_DISTRO: noetic,  ROS_REPO: testing}
+          - {ROS_DISTRO: noetic,  ROS_REPO: main}
+    runs-on: ubuntu-latest
```

Comment:
According to https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners `ubuntu-latest` currently refers to `ubuntu-18.04`.
Maybe it will work regardless, because the industrial ci uses another docker container itself?

Reply:
Done.